src/any_llm/providers/otari/otari.py

```
@@ -227,6 +262,35 @@ async def _chunk_iterator() -> AsyncIterator[ChatCompletionChunk]:
+        """
+        api_kwargs = params.model_dump(exclude_none=True)
+        api_kwargs.pop("stream", None)
+        api_kwargs.update(kwargs)
+
+        if params.stream:
+            return self._stream_messages_async(**api_kwargs)
+
+        response = await self.otari_client.message(**api_kwargs)
+        return MessageResponse.model_validate(_as_plain_dict(response))
+
+    async def _stream_messages_async(self, **api_kwargs: Any) -> AsyncIterator[MessageStreamEvent]:
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**api_kwargs`

(ANN401)